pinakes/main/analytics/analytics_collectors.py

```diff
@@ -0,0 +1,864 @@
+import os
+
+import platform
+import distro
+from django.utils.timezone import now
+from django.utils.translation import gettext_lazy as _
+
+from insights_analytics_collector import CsvFileSplitter, register
+
+from pinakes.main.analytics.collector import AnalyticsCollector
+from pinakes.main.analytics.package import Package
+from pinakes.main.approval.models import Request
+from pinakes.main.catalog import models
+from pinakes.main.common.models import Group
+from pinakes.main.inventory.models import ServiceInventory
+
+
+@register(
+    "config",
+    "1.0",
+    description=_("General platform configuration."),
+    config=True,
+)
+def config(since, **kwargs):
+    # TODO:
+    # license_info = get_license()
+    license_info = {}
+    install_type = "traditional"
+    if os.environ.get("container") == "oci":
+        install_type = "openshift"
+    elif "KUBERNETES_SERVICE_PORT" in os.environ:
+        install_type = "k8s"
+    return {
+        "platform": {
+            "system": platform.system(),
+            "dist": distro.linux_distribution(),
+            "release": platform.release(),
+            "type": install_type,
+        },
+        # 'install_uuid': settings.INSTALL_UUID,
```

Comment:
@hsong-rh @mkanoor I think it'd be good to generate the some INSTALL_UUId here, maybe we'd need to store it into the DB? So the Analytics has some identifier it can work with.

And It looks like we didn't finish a brainstorm on how to get the catalog_url_base. Although this one would be needed only when we have user facing reports (so we can link the user to the catalog instance/page)

Reply:
@Ladas Should we asking the end user for the catalog_url_base since only the customer might now how to access it, since its possible that you might be behind a firewall and not access the catalog server from the cloud.